std/experimental/allocator/building_blocks/ascending_page_allocator.d

```diff
@@ -1,3 +1,7 @@
+// Written in the D programming language.
+/**
+Source: $(PHOBOSSRC std/experimental/allocator/building_blocks/_ascending_page_allocator.d)
+*/
```

Comment:
Meh this is ugly:

![image](https://user-images.githubusercontent.com/4370550/34656649-7d3f02d2-f41d-11e7-8b03-a2f52a4b4858.png)

![image](https://user-images.githubusercontent.com/4370550/34656654-8b399168-f41d-11e7-9704-13080e263f51.png)


http://dtest.dlang.io/artifact/website-110116a6991ae150e89b6abd767a492de920c581-a89969b4d0a4227eed98a2eb0190bd10/web/phobos-prerelease/std_experimental_allocator_building_blocks_ascending_page_allocator.html

I guess I will remove exposing the module via Ddoc from this PR.

CC @Darredevil

Reply:
-> https://github.com/dlang/phobos/pull/6009